en/docs/reference/synapse-properties/scopes.md

```diff
@@ -0,0 +1,158 @@
+
+    ``` java
+    public boolean mediate(org.apache.synapse.MessageContext mc) {  
+        org.apache.axis2.context.MessageContext axis2MsgContext;  
+        axis2MsgContext = ((Axis2MessageContext) mc).getAxis2MessageContext();   
+        // Available only in the sequence the property is defined.  
+        String propValue = (String) axis2MsgContext.getProperty("PropName");  
+        System.out.println("SCOPE_AXIS2 : " + propValue);  
+        return true;  
+    } 
+    ```
+
+## `transport` scope
+
+When the scope of a property is `transport`,
+it refers to the transport headers of the message. [HTTP Transport Properties]({{base_path}}/reference/mediators/property-reference/http-transport-properties/) are available in the axis2 scope.
+
+You can retrieve properties within the
+`transport` scope using the following syntax.
+
+`get-property('transport', String propertyName)`
+
+Examples:
+``` xml
+<property name="PROPERTY_1" value="VALUE_1" scope="transport" type="STRING"/>
+<property name="PROPERTY_2" expression="get-property('transport',PROPERTY_1')"/>
+```
+
+## `registry` scope
+
+The [registry]({{base_path}}/get-started/key-concepts/#registry) in WSO2 MI allows users to store and manage various resources, such as configuration files, XML schemas, WSDL files, XSLT stylesheets, and other reusable assets.
+When the scope of a property is `registry`,
+it refers to a [registry resource]({{base_path}}/develop/creating-artifacts/creating-registry-resources) or a registry resource property.
+
+You can retrieve properties within the registry using the following syntax.
+
+`get-property('registry', String registryPath@propertyName)`  
+`get-property('registry', String registryPath)`
+
+
+Examples:
+``` xml
+<property name="regProperty" expression="get-property('registry', 'gov:/data/xml/collectionx')"/>
+<property name="regResourceProperty" expression="get-property('registry', 'gov:/data/xml/collectionx@abc')"/>
+```
+
+## `system` scope
+
+When the scope of a property is `system`,
+it refers to [Java System properties](https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html).
+You can retrieve properties in system scope using the following syntax.
+
+`get-property('system', String propertyName)`
+
+## `environment` scope
+
+You can retrieve environment variables using the following syntax.
+
+`get-property('env', String propertyName)`
+
+## `file` scope
+
+You can retrieve properties defined in the `file.properties` configuration file using the following syntax. Properties in the file are reloaded periodically according to the time interval defined by the `file.properties.sync.interval` system property in seconds. If this interval is not defined, changes to the properties will not be reloaded automatically during runtime.
+
+`get-property('file', String propertyName)`
+
+## `operation` scope
+
+You can retrieve a property in the operation context level from the
+`         operation        ` scope. The properties within
```

Comment:
```suggestion
`operation` scope. The properties within
```

Shall we fix the formatting here? There are multiple occurrences. Also, let's not break the lines like this. Pls check all the other places as well.